.prettierrc

```diff
@@ -1,15 +1,81 @@
 {
-	"plugins": ["prettier-plugin-apex"],
+	"arrowParens": "avoid",
+	"bracketSpacing": true,
+	"bracketSameLine": true,
+	"printWidth": 100,
+	"semi": true,
+	"singleQuote": true,
 	"trailingComma": "none",
+	"endOfLine": "lf",
+	"apexInsertFinalNewline": false,
 	"useTabs": true,
 	"overrides": [
 		{
 			"files": "**/lwc/**/*.html",
 			"options": { "parser": "lwc" }
 		},
+		{
+			"files": "*.{cls,trigger}",
+			"options": { "parser": "apex", "tabWidth": 2, "useTabs": true }
+		},
+		{
+			"files": "*.trigger",
+			"options": { "printWidth": 200 }

```

Comment:
Why is this different than the other print width specification

Reply:
When the width is less than 200, all the trigger contexts listed go one above each other on separate lines, making it unnecessarily harder (at least for me) to understand. However, since there are no triggers in this project, I can remove it.